.github/workflows/benchexec.yml

```diff
@@ -145,13 +145,13 @@ jobs:
         run: mv $HOME/output.zip ./output.zip      
       - name: Debug Info
         run: ls
-      - uses: actions/upload-artifact@v1
+      - uses: actions/upload-artifact@v4
```

Comment:
Can you do a quick benchexec run? Last time that I tried the local runner script was not smart enough to deal with the proxies.

Reply:
This is indeed still the case.
But, let's continue this conversation here:
https://github.com/esbmc/esbmc/pull/1884